lesson_05/homework_05.py

```diff
@@ -1,34 +1,69 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
+#big_list = [3, 2, 1, 4]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
 # task 1. Знайдіть всі унікальні елементи в списку small_list
-
+print("task 1, унікальні елементи в списку -", set(small_list))
 # task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
-
+summ = 0
+for i in big_list:
+    summ = summ + i
+print("task 2, середнє арифметичне списку big_list ", summ/len(big_list))
```

Comment:
Можна обійтися без циклу
```suggestion
print("task 2, середнє арифметичне списку big_list ", sum(big_list)/len(big_list))
```